src/goods.ts

```diff
@@ -50,9 +55,13 @@ export function sleep(duration: Duration) {
   })
 }
 
-export async function fetch(url: RequestInfo, init?: RequestInit) {
-  $.log({ kind: 'fetch', url, init })
-  return nodeFetch(url, init)
+export async function globalFetch(url: RequestInfo, init?: RequestInit) {
+  if (isNodeEighteenOrGreaterThanEighteen) {
```

Comment:
instead of checking version, why not just check if fetch exists? 

`if ('fetch' in globalThis)` or `if (globalThis.fetch)`

Reply:
@midzelis  Thank you for the feedback. Updated that condition. Please take a look and let me know if you have any doubts.